### PR TITLE
Point devs to how to set args when launching

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
Not sure if the formatting will work for comments. But here it is :)

cc @devoncarew

File: lib/flutter/flutter_launch.dart [hunk 1]
```diff
@@ -70,7 +70,7 @@ class FlutterLaunchType extends LaunchType {
   }
 
   String getDefaultConfigText() {
-    return 'checked: true\ndebug: true\nroute:\n';
+    return 'checked: true\ndebug: true\nroute:\n# additional args for the flutter run command\nargs:\n';
```
Comment (devoncarew) [inline lib/flutter/flutter_launch.dart]: ==> `# Additional args for the flutter run command:`

Also, we may want to break this into a multi-line string for readability (now that it's gotten a bit longer).

**Comment by devoncarew** [comment]
Thanks! One slight tweak to the text.

**Comment by sethladd** [comment]
fixed!

**Pull request merged**